Add alert for checking apps and alarm

longlg88 opened the pull request:
코러산트 환경에서 운영되는 
- Pod가 5분 이상 서비스 불능 상태인 경우
- Pod의 Node가 서비스 불능 상태인 경우
- Pod의 App들의 서비스가 평균 응답시간이 5분동안 5초를 넘는 경우
- Max HPA 값을 사용하고 있는 Pod 확인

추가로 group interval을 3h --> 8h 수정(빈번한 알람 방지) // 이는 환경마다 변경이 되어야 함.

Commit:
helm region will be current region, question for BASE_DOMAIN

longlg88 committed:
add alert for checking apps and alarm

gelius7 reviewed (changes requested):
이번 수정에 각 알람마다 클러스터 이름이 prefix 로 나오기 바랍니다.
ex) [OKC2] Node xxxx is failed. Check node!!
어떤 클러스터에서 발생한 알람인지 알고 싶습니다.

probot-auto-merge[bot] committed:
Merge pull request #378 from opsnow/jj

helm region will be current region, question for BASE_DOMAIN

Commit:
ISTIO install, kiali, jaeger service open

 - ISTIO_DOMAIN config
 - virtual service yaml file for kiali, jaeger
 - istio default gateway values

probot-auto-merge[bot] committed:
Merge pull request #380 from opsnow/issue-istio-1.1.7-kiali-jaeger-open

ISTIO install, kiali, jaeger service open

longlg88 committed:
modify PodCrashingLooping - previous version is wrong

Commit:
add a label for cluster

Commit:
Merge branch 'feat-alertmanager-for-apps' of https://github.com/opsnow/kops-cui

Commit:
Add Cluster name

gelius7 reviewed (approved):
확인했습니다.

pakisto reviewed (approved):
commit
